src/tools/clippy/clippy_utils/src/qualify_min_const_fn.rs

```diff
@@ -18,7 +18,11 @@ use std::borrow::Cow;
 
 type McfResult = Result<(), (Span, Cow<'static, str>)>;
 
-pub fn is_min_const_fn<'a, 'tcx>(tcx: TyCtxt<'tcx>, body: &'a Body<'tcx>, msrv: Option<RustcVersion>) -> McfResult {
+pub fn is_min_const_fn<'a, 'tcx>(
```

Comment:
hm, this file got auto-formatted I think...

Reply:
Yeah.. There were a couple small changes to make this compile with the new `ty::Dynamic` field, but my editor is set up to reformat everything on save. This should be fixed now.